Sources/SwiftSDKCommand/SwiftSDKInstall.swift

```diff
@@ -73,6 +77,17 @@ struct InstallSwiftSDK: SwiftSDKSubcommand {
                 .throttled(interval: .milliseconds(300))
         )
 
+        let bundlePathOrURL = if let experimentalAlias {
+            try SwiftToolchainVersion(
+                toolchain: hostToolchain,
+                fileSystem: self.fileSystem
+            ).urlForSwiftSDK(aliasString: experimentalAlias)
+        } else if let bundlePathOrURL {
+            bundlePathOrURL
+        } else {
+            throw InternalError("foo")
```

Comment:
This should be changed to a real message before merging.

Reply:
Sorry I missed that, I've added a separate error type for this. I hope this addresses all of the remaining concerns.